query-coordinator/src/main/scala/com/socrata/querycoordinator/QueryRewriter.scala

```diff
@@ -424,7 +425,8 @@ class QueryRewriter(analyzer: SoQLAnalyzer[SoQLType]) {
     }
 
     analysisMap.foreach {
-      case (k, Failure(e)) => log.warn(s"Couldn't parse rollup '${rollupMap.get(k).get}'", e)
+      case (rollupName, Failure(e: SoQLException)) => log.info(s"Couldn't parse rollup $rollupName, ignoring: ${e.toString}")
```

Comment:
I want it to include the exception name as well as the message because not
all messages are obvious without the context of what type of exception it
was... and toString is handily defined to do just that unless overridden
specifically, and this isn't an externally visible thing that needs any
sort of sanitization.

On Tue, Mar 29, 2016 at 12:35 PM, Kaida Masaki <notifications@github.com>
wrote:

> In
> query-coordinator/src/main/scala/com/socrata/querycoordinator/QueryRewriter.scala
> <https://github.com/socrata-platform/query-coordinator/pull/66#discussion_r57786214>
> :
>
> > @@ -424,7 +425,8 @@ class QueryRewriter(analyzer: SoQLAnalyzer[SoQLType]) {
> >      }
> >
> >      analysisMap.foreach {
> > -      case (k, Failure(e)) => log.warn(s"Couldn't parse rollup '${rollupMap.get(k).get}'", e)
> > +      case (rollupName, Failure(e: SoQLException)) => log.info(s"Couldn't parse rollup $rollupName, ignoring: ${e.toString}")
>
> Should this be .getMessage instead of .toString?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/socrata-platform/query-coordinator/pull/66/files/557c271e84d936e56792dad993cac69af697b24f#r57786214>
>